server/authentication/passport.ts

```diff
@@ -66,6 +66,12 @@ export default function passportSetup(app: Application, hmppsAuthService: HmppsA
 
           verified(null, loggedInUser)
         } catch (error) {
+          // It almost certainly is an Error, but we need to convince the compiler
```

Comment:
what actually happens in the framework if we don't call the verified callback with the error?

could it be better to just cast the error to an `Error` and pass it anyway?

Reply:
Or we could create our own `Error` explaining what's gone wrong and the non-`Error` value we caught?

Reply:
Hang on, wait, that's exactly what I've done. I reckon this approach is alright - it's unlikely to occur and we'll still describe the value we caught.